ReSharper.FSharp/src/FSharp.Psi.Features/src/Intentions/FunctionAnnotationAction.fs

```diff
@@ -99,7 +99,8 @@ type FunctionAnnotationAction(dataProvider: FSharpContextActionDataProvider) =
         isAtLetExprKeywordOrNamedPat dataProvider letBindings && not (isAnnotated bindings.[0])
 
     override x.ExecutePsiTransaction _ =
-        let binding = dataProvider.GetSelectedElement<IBinding>()
+        let letBindings = dataProvider.GetSelectedElement<ILetBindings>()
+        let binding = letBindings.Bindings |> Seq.exactlyOne
```

Comment:
Wouldn't it fail on the following?
```fsharp
let rec f _ = ()
and g _ = ()
```

Reply:
Ah, it's cheched before, great.

Reply:
Yeah, just confirmed as much myself, feels like this is sufficient for what's currently available.